src/java/org/apache/cassandra/metrics/ColumnFamilyMetrics.java

```diff
@@ -426,9 +430,12 @@ public Double getValue()
         coordinatorScanLatencyByCL = MetricUtils.byConsistencyLevel(cl -> new LatencyMetrics(factory, "CoordinatorScan", ",consistency=" + cl, coordinatorScanLatency));
 
         blockingReadRepairLatency = new LatencyMetrics(factory, "BlockingReadRepair", new LatencyMetrics(globalNameFactory, "BlockingReadRepair"));
+        avoidedReadRepairs = Metrics.meter(factory.createMetricName("AvoidedReadRepairs"));
         blockingReadRepairs = Metrics.meter(factory.createMetricName("BlockingReadRepairs"));
         backgroundReadRepairs = Metrics.meter(factory.createMetricName("BackgroundReadRepairs"));
         attemptedReadRepairs = Metrics.meter(factory.createMetricName("AttemptedReadRepairs"));
+        digestReads = Metrics.meter(factory.createMetricName("DigestReads"));
+        dataReads = Metrics.meter(factory.createMetricName("DataReads"));
```

Comment:
Can you add those panels to: https://grafana.palantircloud.com/grafana/d/ceg4st0zdq41sc/23-pds-6618933a-read-repairs?orgId=1&from=now-6h&to=now&timezone=browser&var-apolloenvironmentid=oz-production&var-cluster=foundry&var-host=$__all?